### PR TITLE
Use built-ins, updated API routes

### PR DESCRIPTION
**Pull request by correaswebert**
* `next/head` provides the functionalities provided by `React Helmet`.
  So no need for an additional dependency.
* The favicon was named `RDSLogo.png`, so renamed it from the
  default of `favicon.ico`.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,2 @@
-NEXT_PUBLIC_BASE_URL='https://staging-api.realdevsquad.com'
+NEXT_PUBLIC_BASE_URL='https://api.realdevsquad.com'
```
Comment (whyDontI) [inline .env:1]: Backend team will be hosting staging API soon, we don't need to change all the env files.
We want the development environment to use staging API.

We can revert this change.

File: .env.development [hunk 2]
```diff
@@ -1,2 +1,2 @@
-NEXT_PUBLIC_BASE_URL='https://staging-api.realdevsquad.com'
+NEXT_PUBLIC_BASE_URL='https://api.realdevsquad.com'
```
Comment (whyDontI) [inline .env.development:1]: Not needed as explained in previous comment

**Review by whyDontI — DISMISSED** [review]
Good catch 👍🏽 
Write a good description for this PR, explaining the changes you did and the reason.
Please create an issue and mention the issue in the description of this PR.

Requested some little changes, everything else looks good 👍🏽 🥳

**Review by swarajpure — COMMENTED** [review]
If we're not at all going to use `react-helmet`, can we remove that package from the `package.json?`

File: .env (hunk 1 above)
Comment (correaswebert) [reply .env:1]: Okay! Will revert 👍🏾

**Comment by correaswebert** [comment]
> If we're not at all going to use `react-helmet`, can we remove that package from the `package.json?`

**Review by whyDontI — APPROVED** [review]
Good work @correaswebert 👍🏽
